s64da_benchmark_toolkit/reporting.py

```diff
@@ -122,8 +122,9 @@ def _save_explain_plan(self, query_metric):
             plan_file.write(query_metric.plan)
 
     def _save_prepare_metrics(self):
-        if os.path.exists(self.prepare_metrics_path):
-            copyfile(self.prepare_metrics_path, os.path.join(self.results_root_dir, self.prepare_metrics_path))
+        if not os.path.exists(self.results_root_dir):
+            os.mkdir(self.results_root_dir) 
+        copyfile(self.prepare_metrics_path, os.path.join(self.results_root_dir, self.prepare_metrics_path))
```

Comment:
`prepare_metrics_path` is a misleading name, since this is not really a path but rather a file name. This makes `os.path.join(self.results_root_dir, self.prepare_metrics_path)` strange to read - I had to go the the definition to see what it's all about.

Reply:
How about simply renaming it to eg. prepare_metrics_filename ?